packages/core/src/interaction-output.ts

```diff
@@ -78,6 +80,11 @@ export class InteractionOutput implements WoT.InteractionOutput {
     }
 
     async value<T extends WoT.DataSchemaValue>(): Promise<T> {
+        // is there any value expected at all?
+        if (this.schema == null) {
+            return this.#value as T;
```

Comment:
At this point, `this.#value` would be `undefined`, right? Maybe we could return that value explicitly to make it more obvious? However, this might not make that much sense when it comes to the cast to `T` 🤔

```suggestion
            return undefined as T;
```

Reply:
Exactly that was the problem...  hence I went the _easy_ way 😉.

![image](https://github.com/eclipse-thingweb/node-wot/assets/11281461/ed07b912-e193-415b-9aa5-77a9b5035518)

Maybe we can just add a comment that it returns `undefined` ?

Reply:
> Exactly that was the problem... hence I went the easy way 😉.

Oh, I see!

> Maybe we can just add a comment that it returns undefined ?

Sure, maybe something like this? :)

```suggestion
            // #value is undefined at this point
            return this.#value as T;
```

Reply:
@JKRhb I made returning undefined more explicitly, see https://github.com/eclipse-thingweb/node-wot/pull/1230/commits/04d96864bfc41c413d1da2c15e9eeb880d0d2830
It doesn't look nice either 🙈